rio_alpha/findnodata.py

```diff
@@ -89,7 +91,7 @@ def discover_ndv(rgb_orig, debug, verbose):
         return output
 
     else:
-        return "Invalid %s " % (str(candidate_list))
+        raise ValueError("Invalid candidate list {!r}".format(candidate_list))
```

Comment:
We should always raise exceptions instead of returning a string explaining the error.